Templates/make_confluence_EC2-node.tmplt.json

```diff
@@ -210,10 +210,18 @@
         "t2.small",
         "t2.medium",
         "t2.large",
+        "t3.micro",
+        "t3.small",
+        "t3.medium",
+        "t3.large",
         "c4.large",
         "c4.xlarge",
         "m4.large",
         "m4.xlarge"
+        "m5.large"
```

Comment:
Please note that TravisCI is failing on your job (the red x next to this commit in the commit list). If you look at the Travis report, you'll see that it's failing on the syntax-checking for this file. There are errors on this line (221) and the next two lines (222-223) that need to be corrected.